pkg/detector/ospkg/oracle/oracle_test.go

```diff
@@ -151,6 +151,95 @@ func TestScanner_Detect(t *testing.T) {
 			},
 			want: nil,
 		},
+		{
+			name:     "the correct installed version without ksplice2",
+			fixtures: []string{"testdata/fixtures/oracle8.yaml"},
+			args: args{
+				osVer: "8",
+				pkgs: []ftypes.Package{
+					{
+						Name:       "glibc",
+						Version:    "2.28",
+						Release:    "151.0.1.el8",
+						Arch:       "x86_64",
+						SrcName:    "glibc",
+						SrcVersion: "2.28",
+						SrcRelease: "151.0.1.el8",
+					},
+				},
+			},
+			want: nil,
+		},
+		{
+			name:     "the correct installed version with ksplice2",
+			fixtures: []string{"testdata/fixtures/oracle8.yaml"},
+			args: args{
+				osVer: "8",
+				pkgs: []ftypes.Package{
+					{
+						Name:       "glibc",
+						Epoch:      2,
+						Version:    "2.28",
+						Release:    "151.0.1.ksplice2.el8",
+						Arch:       "x86_64",
+						SrcEpoch:   2,
+						SrcName:    "glibc",
+						SrcVersion: "2.28",
+						SrcRelease: "151.0.1.ksplice2.el8",
+					},
+				},
+			},
+			want: nil,
+		},
+		{
+			name:     "the installed version has ksplice1",
+			fixtures: []string{"testdata/fixtures/oracle8.yaml"},
+			args: args{
+				osVer: "8",
+				pkgs: []ftypes.Package{
```

Comment:
We already have the same test for ksplice1. These tests are duplicate. We should add only 1 test where the installed version has ksplice2.